inlineplz/interfaces/swarm.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+
+import requests
+import random
+
+from inlineplz.interfaces.base import InterfaceBase
+
+class SwarmInterface(InterfaceBase):
+    def __init__(self, username, password, host, topic, version='v8'):
```

Comment:
ok so this is where you wanted to change this to taking some sort of config object that would be created in `main.py` right?